renderer/API/office365API.ts

```diff
@@ -0,0 +1,99 @@
+import { LogLevel } from "@azure/msal-browser";
+
+const PORT = process.env.NEXT_PUBLIC_PORT;
+const CLIENT_ID = process.env.NEXT_PUBLIC_OFFICE365_CLIENT_ID;
+
+export const msalConfig = {
+  auth: {
+    clientId: CLIENT_ID,
+    redirectUri: `http://localhost:${PORT}/settings`,
+  },
+  cache: {
+    cacheLocation: "localStorage",
+    storeAuthStateInCookie: false, // Set this to "true" if you are having issues on IE11 or Edge
+  },
+  system: {
+    loggerOptions: {
+      loggerCallback: (
+        level: LogLevel,
+        message: string,
+        containsPii: boolean
+      ) => {
+        if (containsPii) {
+          return;
+        }
+        switch (level) {
+          case LogLevel.Error:
+            console.error(message);
+            return;
+          case LogLevel.Info:
+            console.info(message);
+            return;
+          case LogLevel.Verbose:
+            console.debug(message);
+            return;
+          case LogLevel.Warning:
+            console.warn(message);
+            return;
+          default:
+            return;
```

Comment:
Perhaps we don't need So much output to the console? :)
![image](https://github.com/ukad-group/timetracker-desktop-client/assets/133204916/20db41a1-d204-4e31-bfc5-0fa426936644)

Reply:
This is probably a separate task. But it would be nice to remove repetitions between Google calendar and Office 365 calendar
![image](https://github.com/ukad-group/timetracker-desktop-client/assets/133204916/51cf69a3-ee76-48ad-b1ea-96b77611fb66)

Reply:
let it be another task